appveyor.yml

```diff
@@ -106,6 +106,16 @@ install:
   - cmd: ECHO "Installed SDKs:"
   - ps: "ls \"C:/Program Files/Microsoft SDKs/Windows\""
 
+  # Download visual-studio environment script from ogrisel/python-appveyor-demo
+  - ps: "\
+      (New-Object Net.WebClient).DownloadFile(\
+        'https://raw.githubusercontent.com/ogrisel/python-appveyor-demo/\
+          f54ec3593bcea682098a59b560c1850c19746e10/appveyor/run_with_env.cmd', \
+        (
+          Get-Item \".\\ci\\appveyor\\run-with-visual-studio.cmd\" -Verbose\
```

Comment:
The full path is necessary for the download according to this: http://www.appveyor.com/docs/how-to/download-file.

We could download anew, but I kind of like having a stub in the work tree.  It's a good compromise between not having our own version of the file and letting others know that it's still an important part of the testing.

(That and the tests have just finished successfully, and I really want to merge this and be done with it.) :stuck_out_tongue_closed_eyes: